src/pybind/mgr/volumes/fs/volume.py

```diff
@@ -600,6 +606,63 @@ def subvolume_exists(self, **kwargs):
                 ret = self.volume_exception_to_retval(ve)
         return ret
 
+    def get_earmark(self, **kwargs):
+        ret        = 0, "", ""
+        volname    = kwargs['vol_name']
+        subvolname = kwargs['sub_name']
+        groupname  = kwargs['group_name']
+
+        try:
+            with open_volume(self, volname) as fs_handle:
+                with open_group(fs_handle, self.volspec, groupname) as group:
+                    with open_subvol(self.mgr, fs_handle, self.volspec, group, subvolname, SubvolumeOpType.USER_METADATA_GET) as subvolume:
+                        log.info("getting for subvolume %s", subvolume.path)
+                        fs_earmark = CephFSVolumeEarmarking(fs_handle, subvolume.path)
+                        earmark = fs_earmark.get_earmark()
+                        ret = 0, earmark, ""
+        except VolumeException as ve:
```

Comment:
This should also handle EarMarkException.

Reply:
Makes sense! Will add except for handling `EarMarkException` too